Composer/packages/client/src/constants.ts

```diff
@@ -184,6 +190,13 @@ export const addSkillDialog = {
   },
 };
 
+export const repairSkillDialog = (name: string) => {
+  return {
+    title: formatMessage('Link to this skill has been broken'),
+    subText: formatMessage('{name} cannot be found at the location.', { name }),
```

Comment:
`{name} cannot be found at the specified location.`, maybe. I'd ask someone on the design team how to phrase this.

Reply:
Thanks for doing that @beyackle , here is design
https://www.figma.com/file/LlTlh5vXwq91zjGnFtrUUR6l/Composer-main-design-spec-%2F-UI-library?node-id=4511%3A33187